WordPress/Classes/Services/JetpackCapabilitiesService.swift

```diff
@@ -0,0 +1,36 @@
+import WordPressKit
+
+@objc class JetpackCapabilitiesService: NSObject {
+
+    let capabilitiesServiceRemote: JetpackCapabilitiesServiceRemote
+
+    init(capabilitiesServiceRemote: JetpackCapabilitiesServiceRemote? = nil) {
+        let api = WordPressComRestApi.defaultApi(in: ContextManager.shared.mainContext, localeKey: WordPressComRestApi.LocaleKeyV2)
```

Comment:
What do you think about injecting the context in `init`?

Reply:
24226bef15e87767975b6a592cdf1037f3de51b8